### PR TITLE
Initial 'About' page content, #171

### PR DESCRIPTION
**Pull request by knod**

### CODE REVIEW

File: src/AboutContent.js [hunk 1]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
...
+      <p><a href='#learnMore'>Learn more about cliff effects</a></p>
+
+      <Header as='h3'>How do I use this tool?</Header>
+      <p>Go step-by-step to add information about a client's current benefits, household, income, and other relevant information. This information will be used to predict the client's approximate benefit amount. When you reach the end, change the 'Future Income' amount to see how a change in earned income will cause a change in benefit amount. Currently, the SNAP and Section 8 Housing Voucher programs are both available. Note that predictions may not directly match up with a client’s current benefit amount. The app’s focus is the amount of change that occurs in benefits when there are changes in earned income.</p>
+      <p>Please note that this app does not store user data, so if you refresh the page the data you've entered will be lost. Each time you go through the app, it's a clean slate.</p>
```
Comment (shp54) [inline src/AboutContent.js]: Bold this.

File: src/AboutContent.js [hunk 2]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
...
+      <p>This application is part of a project made possible by a Boston Foundation Open Door Grant to the University of Massachusetts Boston's <a href='https://www.umb.edu/csp'>Center for Social Policy</a>, in close partnership with <a href='http://www.prohope.org/'>Project Hope</a> and <a href='http://www.codeforboston.org/'>Code for Boston</a>. The Center for Social Policy is the lead partner for the <a href='https://onsolidgroundma.org/'>On Solid Ground Coalition</a>.</p>
+      <p>The code base is being maintained <a href='https://github.com/codeforboston/cliff-effects'>on GitHub</a> by <a href='http://www.codeforboston.org/'>Code for Boston</a> volunteers. For more information or to report a bug, please contact andrew@codeforboston.org.</p>
+
+      <Header as='h3'>Thank You</Header>
+      <p>Here's a special thank you to all the Code for Boston volunteers who brought you this application, especially Annie LaCourt, Isaac Chansky, Michelle Bernstein, Alec Danaher, Sasha Maryl, Drew Love, Liani Lye, Liam Morley, Nick Francisci, Stephen Chin, Shameek Poddar, Will McIntosh, Don Blair, Ethan Strominger, Nick Lee, Jonathan Marcus, and Emily Wasserman.</p>
```
Comment (shp54) [inline src/AboutContent.js]: Alphabetize this list and link names to LinkedIn/Twitter/Github profiles.

File: src/AboutContent.js [hunk 3]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
...
+      <p>This tool can help show how a change in income affects how much someone receives in public assistance from SNAP and Section 8 Housing Voucher benefits. It was designed for the case managers at <a href='http://www.prohope.org/'>Project Hope</a> with the aim of helping to predict changes in their client's benefits.</p>
+
+      <Header as='h3'>Why are you doing this?</Header>
+      <p>When a slight change in a household’s circumstances - say, a slight pay raise - disproportionately lowers their benefits, that’s called a “cliff effect”. The household is working to increase what they earn, but they end up with a net loss that actually puts them further behind. These “cliff effects” prevent many families from actually getting off of public assistance programs.</p>
+      <p>Cliff effects are also very hard to predict. The interactions between income, household size, many other criteria, and the effects of the programs themselves impact each other in unexpected ways. We're exploring ways to deal with this issue of complexity and help families better understand and predict their situation.</p>
+      <p><a href='#learnMore'>Learn more about cliff effects</a></p>
```
Comment (shp54) [inline src/AboutContent.js]: Fix anchors

File: src/AboutContent.js [hunk 4]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
...
+
+      <Header as='h3'>How do I use this tool?</Header>
+      <p>Go step-by-step to add information about a client's current benefits, household, income, and other relevant information. This information will be used to predict the client's approximate benefit amount. When you reach the end, change the 'Future Income' amount to see how a change in earned income will cause a change in benefit amount. Currently, the SNAP and Section 8 Housing Voucher programs are both available. Note that predictions may not directly match up with a client’s current benefit amount. The app’s focus is the amount of change that occurs in benefits when there are changes in earned income.</p>
```
Comment (ManickYoj) [inline src/AboutContent.js]: Suggestion: add a link here to the client page. I'd suggest inserting it in the first line as 'Go step-by-step on the [client page](#client) ...'

File: src/AboutContent.js (hunk 3 above)
Comment (ManickYoj) [inline src/AboutContent.js]: This link is currently dead - it reroutes to /#/learnMore, which is a non-existent page (that does not 404 for ... I don't know why, and we should fix that separately). Either fix (to route to the header below) or remove for now as we don't introduce any broken links, especially as we close in on a deadline and have other priorities.

File: src/AboutContent.js [hunk 5]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
     <wrapper>
```
Comment (ManickYoj) [inline src/AboutContent.js]: In the AboutPage (above this component level), I'd suggest we center the content box in the row. The width looks reasonable, but the fact that it's left aligned in a larger container is a bit odd.

File: src/AboutContent.js (hunk 1 above)
Comment (ManickYoj) [inline src/AboutContent.js]: May want to emphasize this as a note: I suggest a [message](https://react.semantic-ui.com/collections/message) component block.

File: src/AboutContent.js [hunk 6]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
...
       
       <Header as='h1' style={{ fontSize: '4em' }}>About the Cliff Effects Tool</Header>
-      <p style={{ fontSize: '1.33em' }}>
-        Fill in information about the tool here.
-      </p>
+
+      <Header as='h3'>What is this?</Header>
+      <p>This tool can help show how a change in income affects how much someone receives in public assistance from SNAP and Section 8 Housing Voucher benefits. It was designed for the case managers at <a href='http://www.prohope.org/'>Project Hope</a> with the aim of helping to predict changes in their client's benefits.</p>
+
+      <Header as='h3'>Why are you doing this?</Header>
```
Comment (ManickYoj) [inline src/AboutContent.js]: Bit of an odd phrasing. Sounds like [what the hero says](https://user-images.githubusercontent.com/2252820/33639202-30bda93e-d9f8-11e7-9575-19806ff33d87.png) in a plotline when they're backstabbed by a team member. I'd suggest the less pointed 'Why is this important?'.

File: src/AboutContent.js [hunk 7]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
...
+      <p><a href='#learnMore'>Learn more about cliff effects</a></p>
+
+      <Header as='h3'>How do I use this tool?</Header>
+      <p>Go step-by-step to add information about a client's current benefits, household, income, and other relevant information. This information will be used to predict the client's approximate benefit amount. When you reach the end, change the 'Future Income' amount to see how a change in earned income will cause a change in benefit amount. Currently, the SNAP and Section 8 Housing Voucher programs are both available. Note that predictions may not directly match up with a client’s current benefit amount. The app’s focus is the amount of change that occurs in benefits when there are changes in earned income.</p>
+      <p>Please note that this app does not store user data, so if you refresh the page the data you've entered will be lost. Each time you go through the app, it's a clean slate.</p>
+
+      <Header as='h3'>Who's doing this?</Header>
```
Comment (ManickYoj) [inline src/AboutContent.js]: Alright - in general, these question headers are getting pretty weird. I'd prefer to move away form the 'question word' ... 'this?' pattern. I'd suggest 'Who made this tool?' or some other, more specific phrasing.

File: src/AboutContent.js [hunk 8]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
     <wrapper>
       
       <Header as='h1' style={{ fontSize: '4em' }}>About the Cliff Effects Tool</Header>
-      <p style={{ fontSize: '1.33em' }}>
-        Fill in information about the tool here.
-      </p>
+
+      <Header as='h3'>What is this?</Header>
```
Comment (ManickYoj) [inline src/AboutContent.js]: 'This' is ambiguous. What is this ... about page? header? Suggestion: 'What is this website for?'

File: src/AboutContent.js [hunk 9]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
...
+      <p>Please note that this app does not store user data, so if you refresh the page the data you've entered will be lost. Each time you go through the app, it's a clean slate.</p>
+
+      <Header as='h3'>Who's doing this?</Header>
+      <p>This application is part of a project made possible by a Boston Foundation Open Door Grant to the University of Massachusetts Boston's <a href='https://www.umb.edu/csp'>Center for Social Policy</a>, in close partnership with <a href='http://www.prohope.org/'>Project Hope</a> and <a href='http://www.codeforboston.org/'>Code for Boston</a>. The Center for Social Policy is the lead partner for the <a href='https://onsolidgroundma.org/'>On Solid Ground Coalition</a>.</p>
+      <p>The code base is being maintained <a href='https://github.com/codeforboston/cliff-effects'>on GitHub</a> by <a href='http://www.codeforboston.org/'>Code for Boston</a> volunteers. For more information or to report a bug, please contact andrew@codeforboston.org.</p>
+
+      <Header as='h3'>Thank You</Header>
```
Comment (ManickYoj) [inline src/AboutContent.js]: 'Thank You' is also a bit of an odd header. I'd suggest adding this to the preceding 'who' section.

File: src/AboutContent.js [hunk 10]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
...
+      <p>Here's a special thank you to all the Code for Boston volunteers who brought you this application, especially Annie LaCourt, Isaac Chansky, Michelle Bernstein, Alec Danaher, Sasha Maryl, Drew Love, Liani Lye, Liam Morley, Nick Francisci, Stephen Chin, Shameek Poddar, Will McIntosh, Don Blair, Ethan Strominger, Nick Lee, Jonathan Marcus, and Emily Wasserman.</p>
+
+      <Header as='h3' id={'learnMore'}>Learn More</Header>
```
Comment (ManickYoj) [inline src/AboutContent.js]: I don't think this needs to be in a `{}`. We're not evaluating any JS here.

File: src/AboutContent.js [hunk 11]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
...
+      <p>A bit more about cliff effects:</p>
+      <ul>
+        <li><a href='https://www.youtube.com/watch?v=BveX_rID4_E'>https://www.youtube.com/watch?v=BveX_rID4_E</a></li>
```
Comment (ManickYoj) [inline src/AboutContent.js]: This link should have an informative name, not just a link as text.

File: src/AboutContent.js [hunk 12]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
...
+        <li><a href='http://www.nccp.org/projects/files/NCCP_CO_presentation07.pdf'>http://www.nccp.org/projects/files/NCCP_CO_presentation07.pdf</a></li>
```
Comment (ManickYoj) [inline src/AboutContent.js]: Ditto, informative title.

File: src/AboutContent.js [hunk 13]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
...
+        <li><a href='https://www.umb.edu/editor_uploads/images/centers_institutes/center_social_policy/The_Road_to_the_Cliff_Edge_08.16.17.pdf'>https://www.umb.edu/editor_uploads/images/centers_institutes/center_social_policy/The_Road_to_the_Cliff_Edge_08.16.17.pdf</a></li>
```
Comment (ManickYoj) [inline src/AboutContent.js]: Ditto, informative title.

File: src/AboutContent.js [hunk 14]
```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
...
+      </ul>
+
+      <br/>
+      <p><strong>Important note:</strong> This application is a minimum viable product. It should not be used as the sole tool to understand a client's eligibility for SNAP and Section 8, or for any other public assistance program.</p>
```
Comment (ManickYoj) [inline src/AboutContent.js]: Ditto, message block. Also, if it's important, put it at the top, or ... wait, why is this on a page that users don't have to visit?

Comment (shp54) [reply src/AboutContent.js]: @ManickYoj There's a note to a similar effect on the splash screen at the beginning.

Comment (knod) [reply src/AboutContent.js]: There's also a note in the header on every page. We're just peppering these liberally everywhere.

File: src/AboutContent.js [hunk 15]
```diff
@@ -9,9 +12,36 @@ const AboutContent = function ( props ) {
...
       
       <Header as='h1' style={{ fontSize: '4em' }}>About the Cliff Effects Tool</Header>
-      <p style={{ fontSize: '1.33em' }}>
-        Fill in information about the tool here.
-      </p>
+
+      <Header as='h3'>What is this?</Header>
+      <p>This tool can help show how a change in income affects how much someone receives in public assistance from SNAP and Section 8 Housing Voucher benefits. It was designed for the case managers at <ExternalLink href='http://www.prohope.org/'>Project Hope</ExternalLink> with the aim of helping to predict changes in their client's benefits.</p>
+
+      <Header as='h3'>Why are you doing this?</Header>
```
Comment (ManickYoj) [inline src/AboutContent.js]: Bit of an odd phrasing. Sounds like what the hero says in a plotline when they're backstabbed by a team member. I'd suggest the less pointed 'Why is this important?'.

File: src/AboutContent.js [hunk 16]
```diff
@@ -9,9 +12,36 @@ const AboutContent = function ( props ) {
...
+      <p>When a slight change in a household’s circumstances - say, a slight pay raise - disproportionately lowers their benefits, that’s called a “cliff effect”. The household is working to increase what they earn, but they end up with a net loss that actually puts them further behind. These “cliff effects” prevent many families from actually getting off of public assistance programs.</p>
+      <p>Cliff effects are also very hard to predict. The interactions between income, household size, many other criteria, and the effects of the programs themselves impact each other in unexpected ways. We're exploring ways to deal with this issue of complexity and help families better understand and predict their situation.</p>
+      <p><a href='#about/#learn-more'>Learn more about cliff effects</a></p>
```
Comment (ManickYoj) [inline src/AboutContent.js]: This link is currently dead - it reroutes to /#/learnMore, which is a non-existent page (that does not 404 for ... I don't know why, and we should fix that separately). Either fix (to route to the header below) or remove for now as we don't introduce any broken links, especially as we close in on a deadline and have other priorities.

File: src/AboutContent.js [hunk 17]
```diff
@@ -9,9 +12,36 @@ const AboutContent = function ( props ) {
...
+
+      <Header as='h3'>How do I use this tool?</Header>
```
Comment (ManickYoj) [inline src/AboutContent.js]: May want to emphasize this as a note: I suggest a message component block.

File: src/AboutContent.js [hunk 18]
```diff
@@ -9,9 +12,36 @@ const AboutContent = function ( props ) {
...
+      <p>Go step-by-step to add information about a client's current benefits, household, income, and other relevant information. This information will be used to predict the client's approximate benefit amount. When you reach the end, change the 'Future Income' amount to see how a change in earned income will cause a change in benefit amount. Currently, the SNAP and Section 8 Housing Voucher programs are both available. Note that predictions may not directly match up with a client’s current benefit amount. The app’s focus is the amount of change that occurs in benefits when there are changes in earned income.</p>
+      <p>Please note that this app does not store user data, so <strong>if you refresh the page the data you've entered will be lost</strong>. Each time you go through the app, it's a clean slate.</p>
+
+      <Header as='h3'>Who's doing this?</Header>
```
Comment (ManickYoj) [inline src/AboutContent.js]: Alright - in general, these question headers are getting pretty weird insofar as most would make sense in a horror movie. I'd prefer to move away form the 'question word' ... 'this?' pattern. I'd suggest 'Who made this website?' or some other, more specific phrasing.

File: src/AboutContent.js [hunk 19]
```diff
@@ -9,9 +12,36 @@ const AboutContent = function ( props ) {
...
+      <p>This application is part of a project made possible by a Boston Foundation Open Door Grant to the University of Massachusetts Boston's <ExternalLink href='https://www.umb.edu/csp'>Center for Social Policy</ExternalLink>, in close partnership with <ExternalLink href='http://www.prohope.org/'>Project Hope</ExternalLink> and <ExternalLink href='http://www.codeforboston.org/'>Code for Boston</ExternalLink>. The Center for Social Policy is the lead partner for the <ExternalLink href='https://onsolidgroundma.org/'>On Solid Ground Coalition</ExternalLink>.</p>
+      <p>The code base is being maintained <ExternalLink href='https://github.com/codeforboston/cliff-effects'>on GitHub</ExternalLink> by <ExternalLink href='http://www.codeforboston.org/'>Code for Boston</ExternalLink> volunteers. For more information or to report a bug, please contact andrew@codeforboston.org.</p>
+
+      <Header as='h3'>Thank You</Header>
```
Comment (ManickYoj) [inline src/AboutContent.js]: 'Thank You' is also a bit of an odd header. I'd suggest adding this to the preceding 'who' section.

File: src/AboutContent.js [hunk 20]
```diff
@@ -9,9 +12,36 @@ const AboutContent = function ( props ) {
...
+      <p>Here's a special thank you to all the Code for Boston volunteers who brought you this application, especially Annie LaCourt, Isaac Chansky, Michelle Bernstein, Alec Danaher, Sasha Maryl, Drew Love, Liani Lye, Liam Morley, Nick Francisci, Stephen Chin, Shameek Poddar, Will McIntosh, Don Blair, Ethan Strominger, Nick Lee, Jonathan Marcus, and Emily Wasserman.</p>
+
+      <Header as='h3' name={'learn-more'}>Learn More</Header>
+      <p>A bit more about cliff effects:</p>
+      <ul>
+        <li><ExternalLink href='https://www.youtube.com/watch?v=BveX_rID4_E'>https://www.youtube.com/watch?v=BveX_rID4_E</ExternalLink></li>
```
Comment (ManickYoj) [inline src/AboutContent.js]: This link should have an informative name, not just a link as text.

File: src/AboutContent.js [hunk 21]
```diff
@@ -9,9 +12,36 @@ const AboutContent = function ( props ) {
...
+        <li><ExternalLink href='http://www.nccp.org/projects/files/NCCP_CO_presentation07.pdf'>http://www.nccp.org/projects/files/NCCP_CO_presentation07.pdf</ExternalLink></li>
```
Comment (ManickYoj) [inline src/AboutContent.js]: Ditto, informative title.

File: src/AboutContent.js [hunk 22]
```diff
@@ -9,9 +12,36 @@ const AboutContent = function ( props ) {
...
+        <li><ExternalLink href='https://www.umb.edu/editor_uploads/images/centers_institutes/center_social_policy/The_Road_to_the_Cliff_Edge_08.16.17.pdf'>https://www.umb.edu/editor_uploads/images/centers_institutes/center_social_policy/The_Road_to_the_Cliff_Edge_08.16.17.pdf</ExternalLink></li>
```
Comment (ManickYoj) [inline src/AboutContent.js]: Ditto, informative title.

File: src/AboutContent.js [hunk 23]
```diff
@@ -9,9 +12,36 @@ const AboutContent = function ( props ) {
...
+      </ul>
+
+      <br/>
+      <p><strong>Important note:</strong> This application is a minimum viable product. It should not be used as the sole tool to understand a client's eligibility for SNAP and Section 8, or for any other public assistance program.</p>
```
Comment (ManickYoj) [inline src/AboutContent.js]: Ditto, message block. Also, if it's important, put it at the top.

File: src/AboutContent.js (hunk 14 above)
Comment (ManickYoj) [reply src/AboutContent.js]: Gotcha - let's also pop it to the top of the page, or at least to the 'What ...' section, which is where this is most pertinent.

File: src/AboutContent.js [hunk 24]
```diff
@@ -9,9 +12,36 @@ const AboutContent = function ( props ) {
...
+      <Header as='h3' name={'learn-more'}>Learn More</Header>
+      <p>A bit more about cliff effects:</p>
+      <ul>
+        <li><ExternalLink href='https://www.youtube.com/watch?v=BveX_rID4_E'>https://www.youtube.com/watch?v=BveX_rID4_E</ExternalLink></li>
+        <li><ExternalLink href='http://www.nccp.org/projects/files/NCCP_CO_presentation07.pdf'>http://www.nccp.org/projects/files/NCCP_CO_presentation07.pdf</ExternalLink></li>
+        <li><ExternalLink href='https://www.umb.edu/editor_uploads/images/centers_institutes/center_social_policy/The_Road_to_the_Cliff_Edge_08.16.17.pdf'>https://www.umb.edu/editor_uploads/images/centers_institutes/center_social_policy/The_Road_to_the_Cliff_Edge_08.16.17.pdf</ExternalLink></li>
+      </ul>
+
+      <br/>
+      <Message><strong>Important note:</strong> This application is a minimum viable product. It should not be used as the sole tool to understand a client's eligibility for SNAP and Section 8, or for any other public assistance program.</Message>
```
Comment (ManickYoj) [inline src/AboutContent.js]: Let's also pop this block to the top of the page, or at least to the 'What ...' section, which is where this is most pertinent.

File: src/AboutContent.js (hunk 8 above)
Comment (anhase) [reply src/AboutContent.js]: Yes, let's say "What is this website for?"

File: src/AboutContent.js (hunk 15 above)
Comment (anhase) [reply src/AboutContent.js]: "Why is this tool important?"

File: src/AboutContent.js (hunk 19 above)
Comment (anhase) [reply src/AboutContent.js]: Yes, go ahead and fold into the section above.

File: src/AboutContent.js (hunk 24 above)
Comment (knod) [reply src/AboutContent.js]: Maybe as a 'warning' component or some such as opposed to a 'message' one?

**Comment by lianilychee** [comment]
@knod , feel free to pull my `about-content` branch into yours.  It addresses all the verbiage changes already, so you don't have to.

**Comment by knod** [comment]
@lianilychee: I think once we get your tests passing, we can do that

**Comment by knod** [comment]
I think I may merge this here and we'll deal with merge conflicts on the new PR when we see them.

**Pull request merged**